webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -4723,6 +4727,9 @@ var userOverride = {
                             );
                     }
                     if (email) {
+                        if (!KommunicateUI.validateEmail(email)){
+                            return false;
+                        }  
```

Comment:
If the above regex is correct, then email is always going to be valid. If not, then the end user will see the following message.

![Screenshot 2022-04-01 at 10 32 42 AM](https://user-images.githubusercontent.com/19753380/161198106-a2cd15bd-1e1c-466f-a6d8-71d3c2c6a646.png)

So, I wanted to know is there any possibility that the input pattern validation passed but the `validateEmail()` returns false?

Reply:
"Please match the format requested."
where exactly have we requested a format from the user?

Reply:
@Prochnost55 While testing I tried removing the regex pattern from the Element section of inspect and it passes the validation so have kept validateEmail() for check in this type of case.
But in general if input pattern validation passes then their is no case that validateEmail() returns false.

Reply:
@devashishmamgain Html validates in two way one of it is the default validation in this a proper error message is shown.
![loginError](https://user-images.githubusercontent.com/100116317/161207621-b4aa7fab-2ec1-4248-bbbd-3ad2c75279c5.png)
But it do not validates all types of email such as "something@something" so for this have added a regex pattern to validate, this will give error message like "Please match the format requested."

As the coloum itself says to enter email address so is it known that the format should be a valid email id.

As it is validating by html in browser we cant change the error message, for that we need to remove validation from html and add our at our code.

Reply:
this line will be confusing, is there a way to customize the message?

Reply:
Yes we can pass a customize message, it will be a static message which will be shown in case of wrong email.
![loginerror](https://user-images.githubusercontent.com/100116317/161214140-b10f1ef5-7940-40e4-b159-6887db42ebc4.png)
We can pass "Please enter a valid email address !!".

Reply:
use following:
Please enter a valid email address

Also, add it under MCK_LABELS
https://docs.kommunicate.io/docs/web-localization